robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -158,7 +169,23 @@ function initRosWeb () {
     messageType: 'std_msgs/Float32'
   })
   battery_voltage_listener.subscribe(function (message) {
-    $('#battery-voltage').text(message.data.toFixed(2))
+    let voltage = message.data.toFixed(2)
+    $('#battery-voltage').text(voltage)
+
+    // if statement to control voltage indicator switching between acceptable(white) and unacceptable(red)
+    if ((voltage > MAX_VOLTAGE || voltage < MIN_VOLTAGE) && $('#battery-voltage').attr('acceptable') === '1' && ALERT_ENABLE) {
+      $('#battery-voltage').attr('acceptable', '0')
+      $('#battery-voltage').css({'color': 'red'})
+      errorSound()
+      if (voltage > MAX_VOLTAGE) {
+        navModalMessage('Warning: Voltage too high', 'Disconnect Battery first and then the BMS, and then discharge the Battery to 16.8V')
+      } else if (voltage < MIN_VOLTAGE){
+        navModalMessage('Warning: Voltage too low', 'Turn rover off, disconnect Battery and BMS, and then charge battery to 16.8V')
+      }
+    } else if ($('#battery-voltage').attr('acceptable') === '0' && voltage < MAX_VOLTAGE - VOLTAGE_LEEWAY && voltage > MIN_VOLTAGE + VOLTAGE_LEEWAY) {
+      $('#battery-voltage').attr('acceptable', '1')
+      $('#battery-voltage').css({'color': 'white'})
```

Comment:
so that function doesnt work cause it changes the 'background-color' property instead of the 'color' property.  I'll just create my own function in helpers.js